css/pages/home.css

```
@@ -0,0 +1,181 @@
+body {
+    margin: 0 auto;
+    font-family: -apple-system, BlinkMacSystemFont, "Apple SD Gothic Neo", "Pretendard Variable", Pretendard, Roboto, "Noto Sans KR", "Segoe UI", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
+}
```

❗️ 수정요청
폰트 패밀리 속성을 왜 이렇게 작성하셨을까요? [mdn의 font-family 설명](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#try_it)을 보면 해당 속성에 나열된 폰트명들은 순서대로 우선순위를 가집니다.
즉 위의 값에서는 apple의 시스템 폰트가 우선순위가 제일 높다는 의미로 저희가 사용하고자 했던 Pretendard는 4번째의 우선순위로 앞의 폰트들이 없을때 선택되어집니다.

아래처럼 사용하고자 하는 pretendard 폰트의 우선순위를 높여주시는 것을 추천드려요~

```suggestion
body {
    margin: 0 auto;
    font-family: "Pretendard Variable", Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, "Helvetica Neue", "Segoe UI", "Apple SD Gothic Neo", "Noto Sans KR", "Malgun Gothic", "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
}
```